.github/workflows/ci.yml

```diff
@@ -168,3 +177,4 @@ jobs:
           git config user.email ${{ secrets.GH_BOT_EMAIL }}
           git config user.name ${{ secrets.GH_BOT_NAME }}
           yarn semantic-release
```

Comment:
Note for later: We should look into replacing `semantic-release` with something that understands monorepos. In rnx-kit, [Changesets](https://github.com/changesets/changesets) has been working very well for us.

Reply:
Sweet, will remember that 🙏